android/app/src/main/java/com/dismoi/scout/AccessibilityService/Module.java

```diff
@@ -1,43 +1,41 @@
-package com.dismoi.scout;
+package com.dismoi.scout.AccessibilityService;
 
+import android.content.Context;
+import android.provider.Settings;
+import android.text.TextUtils;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.Callback;
 import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.bridge.ReactMethod;
-import com.facebook.react.bridge.Callback;
-import com.facebook.react.bridge.ReactContext;
 import com.facebook.react.modules.core.DeviceEventManagerModule;
-import android.util.Log;
-import javax.annotation.Nullable;
 
-import android.content.Context;
-import androidx.core.view.accessibility.AccessibilityEventCompat;
-import android.view.accessibility.AccessibilityEvent;
-
-import android.view.accessibility.AccessibilityManager;
-
-import android.text.TextUtils;
-import android.provider.Settings;
-
-import com.facebook.react.bridge.Promise;
+import javax.annotation.Nullable;
 
-public class AccessibilityServiceModule extends ReactContextBaseJavaModule {
+public class Module extends ReactContextBaseJavaModule {
```

Comment:
I'm not sure I understand what's happening here. Is it now a generic module? If so why is it still named `AccessibilityService` in `getName`. And if it's generic is there another service holding accessibility specific code?

Reply:
It's just pure convention. This file is the module in the accessibility service folder (/dismoi/com/scout/AccessibilityService/Module). It could be called AccessibilityServiceModule but I think it's less tiring for the eye to just call it Module. But again... It's pure convention. 
`AccessibilityService` in `getName` is the name of the module called from react. It's related to the name of the folder. You don't seem very seduced by this convention. Do you have other ideas?

Reply:
Nop, I don't have strong feeling about this. Just didn't know about this convention, but it does make sense.